<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>current_setting</class>
 <widget class="QWidget" name="current_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="styleSheet">
        <string notr="true">border:1px solid;
border-radius:20px;
border-width:1px;
border-color: rgb(28, 162, 97);
background-color: rgba(255, 255, 255, 0);
font: 75 14pt &quot;Bell MT&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
